connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -1167,6 +1240,19 @@ export async function githubCodeSyncActivity({
           });
         }
 
+        // Create the folder node
+        await upsertFolderNode({
+          dataSourceConfig,
+          folderId: d.internalId,
+          parents: [
```

Comment:
Just checking: did you check that d.parents did not already contain d.internalId, github-code-$repoId or repoId?

Reply:
Yes, and my local DB state goes in that direction

Reply:
are you sure of that @overmode?
https://github.com/dust-tt/dust/blob/e4eee59268786baf0963230ec02782f815dad1d0/connectors/src/connectors/github/lib/github_api.ts#L884

Reply:
the prod db tells otherwise: `SELECT document_id, parents FROM data_sources_documents WHERE data_source = '1454' AND document_id LIKE 'github-code%';`

Reply:
Will double check

Reply:
Ok seems like the internal id is indeed already there, but not 
`github-code-${repoId.toString()}`, nor `repoId.toString()` judging from the code.

Reply:
Ack, let's update this code then :+1: 
nice catch @aubin-tchoi